### DISCUSSION TITLE
posts/practica-html/

### DISCUSSION THREAD

**Discussion by giscus** [marcosruiz/marcosruiz.github.io#13] [General]
# posts/practica-html/

Entrega y presentación

https://marcosruiz.github.io/posts/practica-html/

**Comment by techdevai-es** [comment]
Aquí no se pone la encuesta de horas, quizá porque es muy lejana y piensas que muchos no se acordarán. Yo las estuve contabilizando pq vi que era muy larga, y porque las indicaciones o cálculos que hacías eran de unas 8 horas para los que sabían menos y unas 4 para los que sabían algo. Sé que compañeros que tenían más nociones la hicieron en 8 horas, a mí me costo unas 16.
